client/src/components/leftPanel/LeftPanel.js

```diff
@@ -22,15 +25,12 @@ class LeftPanel extends Component {
           <img className="logo" src={vacuumLogo} alt="VacuumLabs logo" />
         </a>
         <strong className="sectionTitle">Select tag</strong>
-        <input
-          type="text"
-          value={searchString}
-          onChange={this.handleChange}
-          placeholder="search"
-        />
+        <QuickSearch value={searchString} onChange={this.onSearchChange} />
+        <ShowArchivedBox label="show archived projects" isChecked={showArchived} onChange={this.onShowArchivedChange} />
```

Comment:
According to the issue, this box should be at the bottom.
But I didn't find a way to prevent the tags from going under the box,
and putting it on the top feels more consistent, as the controls are in one place.
![bottom](https://user-images.githubusercontent.com/5972402/74099728-f0433c00-4b26-11ea-8a6c-58fa86a660b1.jpg)
![image](https://user-images.githubusercontent.com/5972402/74099744-2680bb80-4b27-11ea-8fa8-7cb34ffe9bca.png)

Reply:
>  But I didn't find a way to prevent the tags from going under the box,

There is always a way. :slightly_smiling_face: 

What you are saying is not wrong (controls in one place), but I don't think it looks nice and from UX perspective what and how Tomas suggested could work better. Where your solution fails is the search. Search should work with archived projects too always, so it would not be consistent with what the user sees with the checkbox.

He wrote to have an icon/text on the end which would collapse/display archived messages. This could be hidden when you are searching. Also, I believe some visual extinction (an icon? ) for the archived projects would be also nice.